pinot-common/src/main/java/org/apache/pinot/pql/parsers/PinotQuery2BrokerRequestConverter.java

```diff
@@ -120,6 +120,10 @@ private void convertSelectList(PinotQuery pinotQuery, BrokerRequest brokerReques
     List<AggregationInfo> aggregationInfoList = null;
     for (Expression expression : pinotQuery.getSelectList()) {
       ExpressionType type = expression.getType();
+      if (type == ExpressionType.FUNCTION && expression.getFunctionCall().getOperator().equalsIgnoreCase(SqlKind.AS.toString())) {
```

Comment:
What is the SQL standard for `AS` keyword? Can it work on `FUNCTION` and `LITERAL`? If not, we should add sanity check here to block the query. The query should always to valid when it goes to the query engine.

Reply:
It's up to us to set it, here we compile it into a function call with operator = 'AS', operands[0] as alias_from_expression and operands[0] as alias_to_expression